forward43/Mappings.txt

```diff
@@ -0,0 +1,35 @@
+#This file is the mapping file for the Indices we create in Elastic Search.
+
+#1. Create a Index with required number of shards and replicas.
+#2. create the mapping file for the above created Index,
+#3. Add documents into the index.
+
+#creating the index
+PUT /forward43	
+{
+	"settings": {
+		"number_of_shards" : x,   # x is the number of shards
+		"number_of_replicas" : y  # y is the number of replicas
+	}
+}

```

Comment:
There are some optimizations ES has for keywords called normalizers, which you can add into your settings

https://www.elastic.co/guide/en/elasticsearch/reference/current/normalizer.html